articles/virtual-network/virtual-network-scenario-udr-gw-nva.md

```diff
@@ -141,9 +141,13 @@ AZF1 represents an Azure virtual appliance containing the following rules:
 ### AZF2
 AZF2 represents an Azure virtual appliance containing the following rules:
 
-* **Route**: All traffic to 10.0.0.0/16 (**onpremvnet**) must be sent to the Azure gateway IP address (i.e. 10.0.0.1) through **port1**.
 * **Policy**: Allow all bidirectional traffic between **port1** and **port2**.
 
+### AZF3
+AZF3 represents an Azure virtual appliance containing the following rules:
+
+* **Route**: All traffic to 192.168.0.0/16 (**onpremvnet**) must be sent to the Azure gateway IP address (i.e. 10.0.0.1) through **port1**.
```

Comment:
Looking at the diagram, I'm not sure how port1 in AZF3 would be able to talk to Azure Gateway.  Please suggest correction if this is wrong.

Reply:
```suggestion
* **Route**: All traffic to 192.168.0.0/16 (**onpremvnet**) must be sent to the Azure gateway IP address (i.e., 10.0.0.1) through **port1**.
```
Fixing punctuation